### DISCUSSION TITLE
We need to show our works to public

### DISCUSSION THREAD

**Discussion by sykp241095** [ti-community-infra/configs#67] [General]
Hi @ti-community-infra/maintainers ,

As a new sig, we need to follow sig governance policy to show something to public:

1. Roadmap to show our future jobs
2. Timeline to show what happened
3. Weekly report
4. ...

**Comment by 0xPoe** [comment]
cc: @ti-community-infra/sig-community-infra-committers

**Reply by Mini256** [reply]
received.
